src/mediaelement-webaudio.js

```diff
@@ -74,4 +74,28 @@ export default class MediaElementWebAudio extends MediaElement {
 
         this.destroyWebAudio();
     }
+
+    /**
+     * Set the audio volume.
+     *
+     * Previously, calling setVolume on this backend would use the MediaElement backend's setVolume,
+     * which set the volume on the media element. On iOS Safari, media element volume cannot be
+     * programmatically set. Using the gain node allows volume to be changed programmatically.
```

Comment:
> On iOS Safari, media element volume cannot

But I don't see any exception made for Safari so this workaround is safe to apply everywhere then?